tests/validation/test_square_client_id_validation.py

```diff
@@ -22,6 +22,6 @@ def line_data_list(self) -> List[LineData]:
 
     @pytest.mark.skipif(environ.get("CIRCLE_PROJECT_USERNAME") is not None,
                         reason="Server blocking requests from CI server")
-    def test_verify_n(self, line_data_list: pytest.fixture) -> None:
+    def test_verify_p(self, line_data_list: pytest.fixture) -> None:
         validation_result = SquareClientIdValidation.verify(line_data_list)
-        assert validation_result is KeyValidationOption.INVALID_KEY
+        assert validation_result is KeyValidationOption.VALIDATED_KEY
```

Comment:
BTW, how about #181 ?

Reply:
I think mocking is not a good solution for this tests...

I think it would be better to get a response from an actual server.

Reply:
Actually, UT shall use mocking only. 
Tests with wild servers are called "integration", "functional" or something like this. Not "unit-test"

Reply:
I agreed.
Then mocking would be good in this case...